### PR TITLE
List Multiline (example): restore example and style update

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/1050
[Problem] Lack of multiline example
[Solution] restore example and style update

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/list/multiline/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	<div class="ui-page" data-go-to-top-button="true">
+		<header>
+			<h2 class="ui-title">
+				Multiline
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<!-- Adjust it to guide and remove unnecessary sections -->
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/multiline/index.html:27]: I don't think you need any annotations.

File: examples/wearable/UIComponents/contents/presentation-views/list/multiline/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+					<a href="multiline.html">
+						Multiline
+					</a>
+				</li>
+
+				<!-- Add more examples according to guide -->
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/multiline/index.html:33]: ditto.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

**Comment by grzegorz-czajkowski** [comment]
Multiline examples are under clarification (see issue), closing now.

**Pull request closed without merging**